packages/core/contracts/trader/uniswap-broker/UniswapBroker.sol

```diff
@@ -0,0 +1,171 @@
+pragma solidity ^0.6.0;
```

Comment:
Thoughts on putting contracts meant for a DSProxy in their own folder? Or do you see these being useful outside of that context?

Reply:
could do; In the DSProxy client PR: https://github.com/UMAprotocol/protocol/pull/2559 The DSProxy logic was put into `common`. do you have a different pattern in mind? Would you like it all to be in one directory? the way I saw it is that the `DSProxy` stuff is common among the infra but not the contracts. so it might make sence to make a `DSProxy` or `smart-contract-wallet` directory and then we put `trader` as a sub directory. WDYT?